### PR TITLE
Request to merge 

### PR DESCRIPTION
**Pull request by Yczar**
Migrated the project to null safety

### CODE REVIEW

**Comment by JoDeveloper** [comment]
@nickpatrick please merge this request

File: pubspec.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ version: 3.1.1
 homepage: https://github.com/radarlabs/flutter-radar
 
 environment:
-  sdk: ">=2.7.0 <3.0.0"
+  sdk: '>=2.12.0 <3.0.0'
```
Comment (tjulien) [inline pubspec.yaml:7]: Is it necessary to upgrade to `2.12.0`?

Comment (tjulien) [reply pubspec.yaml:7]: Probably this: https://dart.dev/null-safety/unsound-null-safety

**Comment by tjulien** [comment]
Thanks @Yczar ! I just merged your work directly into master. Closing this PR.

**Pull request closed without merging**